cluster/pre_validation.sls

```diff
@@ -15,13 +15,11 @@
   {% endif %}
 {% endif %}
 
-{% if cluster.sbd_checkbox is defined %}
-{% if cluster.sbd.diskless_checkbox is defined and cluster.sbd.diskless_checkbox is sameas true %}
-{% do cluster.sbd.update({'device': false}) %}
-{% endif %}
-{% endif %}
-
-
-{% if cluster.sbd.configure_sbd_checkbox is not defined or cluster.sbd.configure_sbd_checkbox is sameas false %}
-{% do cluster.sbd.pop('configure_resource', none) %}
+{% if cluster.sbd is defined and cluster.sbd_checkbox is defined %}
+  {% if cluster.sbd.diskless_checkbox is defined and cluster.sbd.diskless_checkbox is sameas true %}
+    {% do cluster.sbd.update({'device': false}) %}
+  {% endif %}
+  {% if cluster.sbd.configure_sbd_checkbox is defined and cluster.sbd.configure_sbd_checkbox is sameas false %}
```

Comment:
I think this `cluster.sbd.configure_sbd_checkbox is defined` is not needed, as the `configure_sbd_checkbox` doesn't have the `optional: true` entry, so it will exist always.
I guess it's not harmful anyway

Reply:
yes the above `diskless_checkbox` was optional and I did to match the same.